Changes before release (#5)
aragon/contract-helpers: packages/truffle-5/README.md

# Truffle configuration file

## Set a private key

For interacting with aragonCLI you can configure a private key in `~/.aragon`. Create a file`<network>_key.json`(eg. `rinkeby_key.json`) with this structure:

```json
{
  "rpc": "https://<network>.infura.io",
  "keys": ["put-your-priv-key-here"]
}
```

> Then if you use`--environment aragon:<network>` when using the aragonCLI commands it will use that account.

Reviewer: ```suggestion
aragonCLI will then use that account if you supply`--environment aragon:<network>` to its commands.
```

> You can also define a`~/.aragon/mnemonic.json` file like:

Reviewer: ```suggestion
You can also define a`~/.aragon/mnemonic.json` file instead of a single key, like:
```
